<ui version="4.0" >
 <class>LessOptPageForm</class>
 <widget class="QWidget" name="LessOptPageForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>414</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>-</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="lesson_group" >
     <property name="title" >
      <string>Lesson Descriptions</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QListWidget" name="lessonList" >
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="b_new" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;New...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_modify" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Modify...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_delete" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="b_cleanup" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Clean Up</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>b_new</tabstop>
  <tabstop>b_modify</tabstop>
  <tabstop>b_delete</tabstop>
  <tabstop>b_cleanup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
